builders/tools/eth-dev-env/openzeppelin/defender.md

```diff
@@ -107,13 +107,13 @@ contract PausableBox is Initializable, PausableUpgradeable, OwnableUpgradeable {
 提案指的是合约即将执行的行动。截至本文撰写时，该合约已产生三个主要提案/行动：
 
 - **Pause** —— 检测到暂停功能后可用。可暂停代币转移、铸造及销毁
-- **Upgrade** —— 检测到升级功能后可用。[可通过代理合约对合约进行升级](https://docs.openzeppelin.com/learn/upgrading-smart-contracts)
+- **Upgrade** —— 检测到升级功能后可用。[可通过代理合约对合约进行升级](https://docs.openzeppelin.com/learn/upgrading-smart-contracts){target=blank}
 - **Admin action** —— 可调用受管理合约的任何函数
 
 在本示例中，我们创建了一个新提案，提案内容为暂停合约。为此，需要进行以下操作：
 
- 1. 点击“New proposal（新建提案）”按钮，显示所有选项
- 2. 点击“Pause（暂停）”
+ 1. 点击**New proposal**（新建提案）按钮，显示所有选项
+ 2. 点击**Pause**（暂停）
 
 ![OpenZeppelin Defender Admin Contract New Pause Proposal](/images/builders/tools/eth-dev-env/openzeppelin/oz-defender-4.png)
```

Comment:
this was as close as I could get to the actual link haha but line 125 should be updated to **Create pause proposal**